hibernate-core/src/test/java/org/hibernate/test/extralazy/ExtraLazyTest.java

```diff
@@ -261,5 +261,169 @@ public void testMap() {
 		assertNotNull(child2);
 		session2.close();
 	}
+
```

Comment:
Everyone one of these tests has an identical preparation step.  Pull that out into a common method?